core/settings.py

```diff
@@ -7,11 +7,19 @@ def __init__(self, parent=None):
         self.global_settings = sublime.load_settings("GitSavvy.sublime-settings")
 
     def get(self, key, default=None):
-        project_data = sublime.active_window().project_data()
+        view = sublime.active_window()
+        project_savvy_settings = view.settings().get("GitSavvy", {})
```

Comment:
This change. Where do we set these type of settings?

Reply:
From the `sublime-project` file. The settings in `sublime-project` will be imported to `view.settings()` when the file is loaded.

Reply:
Is that something we make of my default?

Reply:
What i meat was. Do we copy the project settings to the view every time a new view opens?
 if so where?

Reply:
It is done by SublimeText, not by us.

Reply:
Then you can merge it.

Reply:
All good